sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Checks that all tunnels are up
+#
+# Return CRITICAL if any tunnel is down
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+# Carl Hannusch <channusch@us.ibm.com>
+
+import argparse
+import re
+import subprocess
+import sys
+
+STATE_OK = 0
+STATE_CRITICAL = 2
+BACKUP = 1
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-ip', '--ip', help='vyatta vrrp ip',
+                        required=True)
+    parser.add_argument('-p', '--password', help='snmp password',
+                        required=True)
+    args = parser.parse_args()
+
+    status = ""
+    cmd = "snmpwalk -u sensu -A %s -a md5 -l authNoPriv %s \
+           1.3.6.1.2.1.2.2.1.2" % (args.password, args.ip)
```

Comment:
I know I haven't been reviewing these checks otherwise so I might have missed the discussion, but is there a reason we're not using the [`pysnmp` library](http://pysnmp.sourceforge.net/) or some alternative?